librz/core/cmd/cmd_search.c

```diff
@@ -2020,6 +2021,7 @@ static bool do_analysis_search(RzCore *core, struct search_parameters *param, co
 		pj_a(param->pj);
 	}
 	input = rz_str_trim_head_ro(input);
+	rz_return_val_if_fail(input, false);
```

Comment:
i think these should be only set for the input in a function. maybe is better to throw an error.

Reply:
That line is there because otherwise gcc12-asan won't shut up, but yeah if `input` is NULL than `*input` should throw (technically [it's UB](https://en.cppreference.com/w/c/language/operator_member_access) but probably Rizin won't be run on any system where `*NULL` won't throw).